datastore.cloud-ndb.quickstart_test: test_quickstart failed

datastore.cloud-ndb.quickstart_test: test_quickstart failed
buildID: 4aa16fd61986dad63823af87e70fdd9f549604a4
buildURL: https://source.cloud.google.com/results/invocations/3cd496c9-8f13-4fd1-b3f9-b2a8afde1509
status: failed

Build error message:
```
Traceback (most recent call last):
  File "/tmpfs/src/github/python-docs-samples/datastore/cloud-ndb/quickstart_test.py", line 39, in test_quickstart
    assert test_book.title in out
AssertionError: assert '83a90130-bcde-490b-ac7d-aa4e34699608' in ''
 +  where '83a90130-bcde-490b-ac7d-aa4e34699608' = <[ContextError('No current context. NDB calls must be made in context established by google.cloud.ndb.Client.context.') raised in repr()] Book object at 0x7efff41a11d0>.title
```

@andrewsg Do you understand what this error message means and what we need to do to fix it?

--- a/datastore/cloud-ndb/requirements.txt
+++ b/datastore/cloud-ndb/requirements.txt
@@ -1,4 +1,4 @@
 # [START ndb_version]
-google-cloud-ndb==0.1.0
+google-cloud-ndb==1.0.1
 # [END ndb_version]
 flask==1.1.1

